test/system/500-networking.bats

```diff
@@ -911,4 +911,15 @@ EOF
     run_podman network rm $net1
 }
 
+# Issue #20448 - /etc/hostname with --uts=host must show "uname -n"
+@test "podman --uts=host must use 'uname -n' for /etc/hostname" {
+    skip_if_remote "cannot check 'uname -n' on remote"
+    hostname=`uname -n`
+    run_podman run --rm --uts=host $IMAGE cat /etc/hostname
+    assert "$output" = $hostname "/etc/hostname must contain 'uname -n'"
```

Comment:
Failure comment is a little misleading: it's not "contain", it's "be" or "equal".

Also, it's good practice to make failure comments unique. That way a monkey like me, seeing a failure message, can grep for it in the source and find exactly one hit. If there are two hits, monkeys get confused. Perhaps something like this:
```
    ... "/etc/hostname with --uts=host"
....
    ... "/etc/hostname with --net=host and --uts=host"
```